src/mpid/ch4/netmod/ofi/util.c

```diff
@@ -228,7 +232,9 @@ static int MPIDI_OFI_get_huge(MPIDI_OFI_send_control_t * info)
     recv_elem->event_id = MPIDI_OFI_EVENT_GET_HUGE;
     recv_elem->remote_info = *info;
     recv_elem->next = NULL;
-    MPIDI_OFI_get_huge_event(NULL, (MPIR_Request *) recv_elem);
+    if (ready_to_get) {
+        MPIDI_OFI_get_huge_event(NULL, (MPIR_Request *) recv_elem);
+    }
```

Comment:
This block seems fine since it can't be accomplished with the existing `if` block.

Reply:
Yeah, I started the changes here. Then I think there is a benefit in keeping the code style in both places similar. Besides, having the explicit word `ready_to_get` makes the code more self-explainable.